src/Microsoft.Health.Fhir.Api/Features/Routing/UrlResolver.cs

```diff
@@ -330,25 +330,39 @@ public Uri ResolveOperationDefinitionUrl(string operationName)
 
         private Uri GetRouteUri(HttpContext httpContext, string routeName, RouteValueDictionary routeValues, string scheme, string host)
         {
-            try
+            var s = string.Empty;
```

Comment:
it this required?

Reply:
I don't see "s" being used in this method.

Reply:
Removed!